src/main/features/response/views/statement-of-means/monthly-income.njk

```diff
@@ -70,14 +52,24 @@
 
         {% if form.model and form.model.canAddMoreRows() %}
           <div id="add-event-container" class="form-group">
-            {{ linkButton('Add another income', 'action[addRow]', 'button button-secondary') }}
+            {{ linkButton('Add another source of income', 'action[addRow]', 'button button-secondary') }}
           </div>
         {% endif %}
-      </div>
-
-      {{ saveAndContinueButton() }}
+      </div> #}
 
+      {{ saveAndContinueButton() }}  
+      
     </form>
   </div>
 </div>
+
 {% endblock %}
+
+{% macro incomeSource(form, name, label, hint) %}
+  {% set capitalisedName = name.slice(0, 1) | upper + name.slice(1) %}
```

Comment:
nunjucks might be a cleaner way to capitalise the name: 
https://mozilla.github.io/nunjucks/templating.html#capitalize